### DISCUSSION TITLE
Storage link not working

### DISCUSSION THREAD

**Discussion by spartyboy** [laravel/framework#45055] [Q&A]
- Laravel Version: 9.27
- PHP Version: 8.1
- Database Driver & Version:

### Description:
I ran php artisan storage:link on server it created symbolic link correctly but whenever am calling files using asset or storage::url it returns 404, but whenever i set permission to 755 it works and files are found

**Comment by arbazargani** [comment]
try to change owner of script path to user which is runnig script.
like this command: `chown -R USERNAME /var/www/` 
(ofcourse you should replace USERNAME with your actual username, and same for path ...)
